data/oci-migrate-conf.yaml

```diff
@@ -0,0 +1,272 @@
+# ----------------------------------------------------------------------------
+# Configuration and status data with respect to the oci-image-migrate.
+# ----------------------------------------------------------------------------
+#
+# dummy format key
+dummy_format_key : '01234567'
+#
+# python version
+pythonver : 2
```

Comment:
I am not following what we doing here, on OL7 we use python 2 on OL8 we use python 3 - python.six should allow you to not care which one you using...